Packs/ForescoutEyeInspect/IncidentFields/incidentfield-ForescoutEyeInspect_Event_Type.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
@RazOmry We are trying to minimize the creation of new fields. Please use the common field "Event Type" instead.